Config/BuildSettings.swift

```diff
@@ -148,7 +148,7 @@ final class BuildSettings: NSObject {
     
     
     // MARK: -  Public rooms Directory
-    static let publicRoomsShowDirectory: Bool = true
+    #warning("Unused build setting: should this be implemented in ShowDirectory?")
     static let publicRoomsAllowServerChange: Bool = true
```

Comment:
This was only being used in the removed directory, unsure if it is intended functionality for `ShowDirectoryViewController` at some point.